include/geode/basic/attribute_manager.h

```diff
@@ -113,7 +113,7 @@ namespace geode
                     "if an instantiated attribute of the same name "
                     "with different storage already exists." );
 
-                typed_attribute.reset( Attribute< T >::create( args... ) );
+                typed_attribute.reset( new Attribute< T >{ args..., {} } );
```

Comment:
new ?

Reply:
yes, there is issue with make_shared since constructors are not trivials